SDWebImage/SDWebImageImageIOCoder.m

```diff
@@ -160,7 +160,7 @@ - (UIImage *)incrementallyDecodedImageWithData:(NSData *)data finished:(BOOL)fin
         if (partialImageRef) {
             const size_t partialHeight = CGImageGetHeight(partialImageRef);
             CGColorSpaceRef colorSpace = SDCGColorSpaceGetDeviceRGB();
-            CGContextRef bmContext = CGBitmapContextCreate(NULL, _width, _height, 8, _width * 4, colorSpace, kCGBitmapByteOrderDefault | kCGImageAlphaPremultipliedFirst);
+            CGContextRef bmContext = CGBitmapContextCreate(NULL, _width, _height, 8, SDCGByteAlign(_width * 4, 64), colorSpace, kCGBitmapByteOrderDefault | kCGImageAlphaPremultipliedFirst);
```

Comment:
quick question @dreampiggy: are we sure we always run on 64 bit systems? Maybe we can find a way to get that number through a system API call and use it instead of using the the hardcoded value of `64`

Reply:
You're right. This 64 is device architecture related. For ARMv7 it use 32 instead of 64. So I change it to always pass 0 to let Core Graphics automatically calculate do this. See #2119